Sources/Rendering/Core/AbstractMapper3D/index.js

```diff
@@ -16,14 +16,11 @@ function vtkAbstractMapper3D(publicAPI, model) {
     }
   };
 
-  publicAPI.getCenter = (...center) => {
+  publicAPI.getCenter = () => {
```

Comment:
This changes the API. 
It probably shouldn't have just be a patch version release. Let's hope that no-one used the getCenter(center) API externally to VTK.JS :-)

Reply:
Indeed. However, anyone using `getCenter(center)` would have incorrect code anyways, since `getCenter(center)` doesn't actually do anything. (`center` doesn't actually change.)